command/gist.go

```diff
@@ -0,0 +1,78 @@
+package command
+
+import (
+	"fmt"
+	"github.com/cli/cli/api"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(gistCmd)
+	gistCmd.AddCommand(gistCreateCmd)
+	gistCreateCmd.Flags().StringP("description", "d", "", "Required. The filenames and content of each file in the gist. The keys in the files object represent the filename and have the type string.")
+	gistCreateCmd.Flags().StringP("filename", "f", "", "A descriptive name for this gist.")
+	gistCreateCmd.Flags().BoolP("public", "p", false, "When true, the gist will be public and available for anyone to see. Default: false.")
```

Comment:
We usually put the "default" portion in parentheses, e.g. `(default: private)

More precisely, this is what Cobra does for us, but it does not explicitly indicate default values for boolean flags that are false by default.